tests_e2e/orchestrator/lib/agent_test_suite.py

```diff
@@ -565,6 +568,7 @@ def _execute(self) -> None:
 
                     try:
                         test_context = self._create_test_context()
+                        test_args = self._get_test_args()
```

Comment:
we should do the conversion in _initialize() and keep the result in self._test_args, to avoid adding more parameters to the execute method